compute_metrics_reloaded.py

```diff
@@ -246,9 +252,9 @@ def main():
     df_mean = (df.drop(columns=['reference', 'prediction', 'EmptyRef', 'EmptyPred']).groupby('label').
                agg(['mean', 'std']).reset_index())
 
-    # Rename columns
-    df.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
-    df_mean.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
+    # # Rename columns
+    # df.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
+    # df_mean.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
```

Comment:
@naga-karthik, I made all the metrics camel case in https://github.com/ivadomed/MetricsReloaded/pull/6/commits/0b1894a25d1909036a8ec74bcc3cf617ed830026. So maybe we can again use this dict?

Reply:
thanks for this! looks good now